storage/mysql/tree_storage_test.go

```diff
@@ -46,13 +71,36 @@ func nodesAreEqual(lhs []storage.Node, rhs []storage.Node) error {
 	return nil
 }
 
-func TestNodeRoundTrip(t *testing.T) {
-	s, err := NewLogStorage(createLogID(), "test:zaphod@tcp(127.0.0.1:3306)/test")
-	if err != nil {
-		t.Fatalf("Failed to open tree storage: %s", err)
+func createFakeLeaf(db *sql.DB, logID trillian.LogID, hash, data, tsBytes []byte, seq int64, t *testing.T) {
+	_, err := db.Exec("INSERT INTO LeafData(TreeId, LeafHash, TheData) VALUES(?,?,?)", logID.TreeID, hash, data)
+	_, err2 := db.Exec("INSERT INTO SequencedLeafData(TreeId, SequenceNumber, LeafHash, SignedEntryTimestamp) VALUES(?,?,?,?)", logID.TreeID, seq, hash, tsBytes)
+
+	if err != nil || err2 != nil {
+		t.Fatalf("Failed to create test leaves: %v %v", err, err2)
 	}
+}
+
+func checkLeafContents(leaf trillian.LogLeaf, seq int64, hash, data []byte, t *testing.T) {
+	if bytes.Compare(hash, leaf.LeafHash) != 0 {
```

Comment:
I decided to push the boundaries and transcend the limitations of mere arbitrary convention. Done.